pet-store-react-hooks/src/infrastructure/components/LoadingIndicator/LoadingIndicator.tsx

```diff
@@ -24,7 +24,11 @@ export function LoadingIndicator(props: LoadingIndicatorProps) {
   }
 
   return (
-    <div aria-label="loading-indicator" className="spinner-wrapper">
+    <div
+      role="alert"
+      aria-label="loading indicator"
```

Comment:
The label here should be `loading` or `loading pets`  or whatever is loading, not `loading indicator`.